ffxivbot/views/tata.py

```diff
@@ -41,6 +46,7 @@ def generate_bot_conf(bot, client, web_base, http_url, ws_url):
                         "reverseApiPath": "/api",
                         "reverseEventPath": "/event",
                         "useUniversal": True,
+                        "useTLS": False
                         "reconnectInterval": 3000,
                     }
                 ],
```

Comment:
```suggestion
                ],
              },
```

Perhaps??
Won't promiss.